### PR TITLE
Outreachy/MDF task

### PR DESCRIPTION
**Pull request by TaofeeqatDev**
This is the branch I created for my MDF task. In this branch, I created a repository called  Outreachy_Modeci_Projects where I created a separate folder for each of my task. In each folder is a Readme.md file that described the outline of each of this task. I humbly await your feedback on all this tasks. Thanks

### CODE REVIEW

**Comment by pgleeson** [comment]
Thanks for this @TaofeeqatDev.
- For the pytorch to mdf example, please make sure you have run it with the latetst version of mdf, which gives colors to the generated graph of the mdf model. Aslo, use the new solid_color=True option in mdf_model.to_graph_image
- Can you run the model multiple times with different input digits to assess the % accuracy of the MDF model?
- The keras_mdf.py file looks like it comes from a previous commit, which is not in our main branch. Can you be clear where this came from?
- Can you add a new layer to your keras model and retrain it, and demonstrate that the newly generated mdf model also has the new structure? Note you should use these helper methods: https://github.com/ModECI/MDF/blob/main/examples/TensorFlow/Keras/keras2mdf.py

**Comment by TaofeeqatDev** [comment]
@pgleeson thanks for the feedbacks

-  I have tried including solid_color for the pytorch to MDF example but I kept on getting error, that was the reason I removed it.  But I will definitely work on it to fix the error
- The second point is not very clear to me, do you mean I should supply a different input image and try to convert it to MDF again  to see if the pytorch and MDF will still produce the same result.
- As for the third point, I wrote the keras_mdf script myself base on the tutorial I saw in one of the docs, as a matter of fact, I wrote 2 python script (one with 3 layers and another with four) if you want me to send a screenshot of the the second python script for clarification I will gladly do that.
-  As for the last point, I will do that right away and make another commit. Thanks

**Comment by TaofeeqatDev** [comment]
@pgleeson As regards the last feedback on my project, I have added a new layer to my keras model as instructed and have also retrained it and generated the graph. Then i also use the helper function to convert it to MDF. However, everything works perfectly except for the graph. i got the error *parameter object has no attribute shape*, while trying to plot the graph. The parameter object is a list and list objects do not have the attribute *shape*. Looking forward to your feedback
Thanks

**Comment by pgleeson** [comment]
Thanks @TaofeeqatDev.
- It seems though some of your notebooks have disappeared in this version, particularly where the pytorch and keras generated models are converted to mdf.
- Can you remove the files used to generate the images with graphviz from this commit (e.g. AB, example - text files which begin with digraph)?
- Can you tell me where [this file](https://github.com/TaofeeqatDev/MDF/blob/fb6840454248453a060cafedfb34ef655b428b9a/Outreachy_Modeci_Projects/Second_task/keras/keras_mdf.py) came from originally?

**Comment by TaofeeqatDev** [comment]
Thanks @pgleeson 

* I removed the 2 notebooks in order to work on the changes you talked about base on your last feedback.
* I will remove the files from the folder
* I saw the script in one  of the repository I came across while going through some example codes. But since you referred me to another script am already working on that. 
Thanks.

**Comment by TaofeeqatDev** [comment]
@pgleeson  as regards your comment on too many files in PR. My question is, do you want to upload just the two notebook, I mean the notebook for building the model and the one for converting to MDF. Just want to confirm this before I reupload my projects.

**Comment by TaofeeqatDev** [comment]
@pgleeson as regards the last feedback on my task.
* I have added a new layer to my keras model and retrained it.
* I also use the helper functions you referred to for my keras to MDF conversion.
* I'm currently working on the pytorch to MDF conversion and I will reupload it soon.

**Comment by pgleeson** [comment]
@TaofeeqatDev there shouldn't be any reason to remove files and reupload them, e.g. your notebooks. You should be managing files by adding/committing changes using git, rather adding/deleting them on the website.
Please do add the pytorch->mdf version and update that by committing changes to the same file.
Re the keras example, it seems to be just a toy example with dummy data in/out, correct? It would be better if it was a "real world" example, e.g. like the pytorch one 
The file referred to in your keras notebook (`from Keras_MDF import *`) does not exist.

**Comment by TaofeeqatDev** [comment]
> @TaofeeqatDev there shouldn't be any reason to remove files and reupload them, e.g. your notebooks. You should be managing files by adding/committing changes using git, rather adding/deleting them on the website. Please do add the pytorch->mdf version and update that by committing changes to the same file. Re the keras example, it seems to be just a toy example with dummy data in/out, correct? It would be better if it was a "real world" example, e.g. like the pytorch one The file referred to in your keras notebook (`from Keras_MDF import *`) does not exist.

Thanks for the feedback... I will use the MNIST dataset that I use for my pytorch example

**Comment by TaofeeqatDev** [comment]
I have opened a new pull request for my project. Thanks
